apps/balancer-tools/src/app/apr/api/(utils)/validate.ts

```diff
@@ -80,9 +81,11 @@ export const QueryParamsSchema = z
   .object({
     poolId: OptionalNullableString.refine(
       (poolId) =>
-        !poolId ||
         POOLS_WITH_LIVE_GAUGES.some(
           (g) => g.id.toLowerCase() === poolId?.toLowerCase(),
+        ) ||
+        POOLS_WITHOUT_GAUGES.some(
+          (p) => p.id.toLowerCase() === poolId?.toLowerCase(),
```

Comment:
I wasn't able to run on dev 🤔 

<img width="873" alt="image" src="https://github.com/bleu-fi/balancer-tools/assets/63298341/d159a48f-852a-42e8-85ac-a0898eab9ce6">

Reply:
That's odd, cause last deploy's [URL as example](https://tools-monorepo-mrc7mvl0k-bleu-fi.vercel.app/apr/pool/ethereum/0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014?startAt=09-21-2023&endAt=09-22-2023) it seems to work, did u manage to fix this on your end?